packages/daimo-api/src/db/db.ts

```diff
@@ -179,6 +179,14 @@ export class DB {
 
           UNIQUE (address, push_key)
         );
+
+        --
+        -- Jumbo Indexer setup for tracking processed blocks
```

Comment:
i'd call it IndexDB (the nice thing is daimo api doesn't care what is feeding into IndexDB = jumbo or anything else, just that rows show up according to spec)